addons/mail/models/mail_link_preview.py

```diff
@@ -142,3 +148,7 @@ def _gc_mail_link_preview(self):
             ('message_id', '=', False),
             ('create_date', '<', fields.Datetime.now() - timedelta(days=lifetime)),
         ], order='create_date ASC', limit=1000).unlink()
+
+    def _init_ignore_list(self):
```

Comment:
You can actually call that in an xml file, using the "function" tag, which is the standard way of doing it when it is not linked to data migration, especially it allows people to tweak it, override it, ... and it does not require to hack the init.

e.g. (did not test, just for example)
```
<?xml version="1.0" encoding="utf-8"?>
<odoo>
    <data noupdate="1">
        <function model="mail.link.preview" name="_initialize_ignore_list"/>
    </data>
</odoo>
```

Reply:
Much better indeed, I changed to this solution